### PR TITLE
chart samples with value format specifiers

### PR DESCRIPTION
**Pull request by mddifilippo89**
mdd-format-specifiers

### CODE REVIEW

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]
@mddifilippo89 can you add FinChart sample with format specifiers which should be similar to CC sample?

File: samples/charts/category-chart/format-specifiers.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "strings": {
+    "legendTitle": "Highest Grossing Movie Franchises"
+  },
+  "descriptions": {
+    "legend": {
+      "type": "Legend",
+      "name": "Legend",
+      "orientation": "Horizontal"
```
Comment (HUSSAR-mtrela) [inline samples/charts/category-chart/format-specifiers.json]: change legend type to DataLegend and set
      "valueFormatMode": "Currency",
      "valueFormatString": "${0} Billion",

File: samples/charts/category-chart/format-specifiers.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+          "currencyDisplay": "symbol",
+          "minimumFractionDigits": 0
+        }
+      ]
+    }
+  },
+  "modules": [
+    "withDescriptions",
+    "layouts/PropertyEditorPanelModule",
+    "charts/LegendModule",
```
Comment (HUSSAR-mtrela) [inline samples/charts/category-chart/format-specifiers.json]: used DataLegendModule

File: samples/charts/data-chart/format-specifiers.json [hunk 3]
```diff
@@ -0,0 +1,83 @@
+{
+  "strings": {
+    "legendTitle": "Highest Grossing Movie Franchises"
+  },
+  "descriptions": {
+    "legend": {
+      "type": "Legend",
+      "name": "Legend",
+      "orientation": "Horizontal"
```
Comment (HUSSAR-mtrela) [inline samples/charts/data-chart/format-specifiers.json]: change legend type to DataLegend and set
"valueFormatMode": "Currency",
"valueFormatString": "${0} Billion",

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

File: samples/charts/category-chart/format-specifiers.json (hunk 1 above)
Comment (mddifilippo89) [reply samples/charts/category-chart/format-specifiers.json]: done

File: samples/charts/category-chart/format-specifiers.json (hunk 2 above)
Comment (mddifilippo89) [reply samples/charts/category-chart/format-specifiers.json]: updated

File: samples/charts/data-chart/format-specifiers.json (hunk 3 above)
Comment (mddifilippo89) [reply samples/charts/data-chart/format-specifiers.json]: done

**Review by HUSSAR-mtrela — APPROVED** [review]

**Pull request merged**